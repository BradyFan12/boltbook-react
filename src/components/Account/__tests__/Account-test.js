import React from 'react';
import renderer from 'react-test-renderer';
import { Account } from '../';

describe('Account', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Account />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});