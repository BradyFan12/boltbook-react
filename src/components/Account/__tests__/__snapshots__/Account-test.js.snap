// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account renders correctly 1`] = `
<div
  className="container-fluid"
/>
`;
